src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -747,6 +749,43 @@ public void buttonDeleteCvrFileClicked() {
         .removeAll(tableViewCvrFiles.getSelectionModel().getSelectedItems());
   }
 
+  /**
+   * Action when "Auto-Load Candidates" clicked.
+   */
+  public void buttonAutoLoadCandidatesClicked() {
```

Comment:
These are some pretty hardcore calculations that can take a while, so you're going to need to `setGuiIsBusy` for this, and likely create a new service to handle it. See `menuItemValidateClicked` for an example.

Otherwise, users can spam the button and cause things to get locked up pretty badly.

Reply:
Neat! Didn't see that before. Done.